jscomp/test/ocaml_parsetree_test.ml

```diff
@@ -20743,18 +20743,18 @@ let eq loc x y =
 (* #directory "+compiler-libs";; *)
 (* #load "ocamlcommon.cma";; *)
 
-let v str = 
-  str  
-  |> Lexing.from_string 
+let v str =
+  str
+  |> Lexing.from_string
   |> Parse.implementation
 
 
 let _ =
-  match v {|let v str = 
-  str  
-  |> Lexing.from_string 
+  match v {|let v str =
+  str
+  |> Lexing.from_string
   |> Parse.implementation
-|} with 
+|} with
   | {Parsetree.pstr_desc =
    Parsetree.Pstr_value (Asttypes.Nonrecursive,
```

Comment:
hmm, trust me, having your editor to trim whitespace will cause your more trouble than it's worth..
there are quite a few cases where source code is not whitespace insensitive -- e,g. our encoding of .bsbuild is whitespace sensitive

Reply:
Ah I see.

The problem is having trimming is the baseline for most contributors. We should adapt _for_ the contributors, not make them adapt to us for one or two bugs potential here (which would have been caused anyway if someone accidentally adds a space)